nuitka/build/Backend.scons

```diff
@@ -825,8 +827,11 @@ def getWindowsPythonDLLPath():
 
 if win_target:
     # Some non standard Pythons on Windows have this.
-    if gcc_mode and static_libpython:
+    if static_libpython:
         env.Append(LIBS=[env.File(static_libpython)])
+        env.Append(LIBPATH=[os.path.join(python_prefix, 'libs')])
+
+        env.Append(LIBS=["User32.lib", "Advapi32.lib", "Ole32.lib", "OleAut32.lib", "Ws2_32.lib", "version.lib", "Iphlpapi.lib", "crypt32.lib", "Rpcrt4.lib", "Shlwapi.lib", "libcrypto.lib", "libssl.lib", "_asyncio.lib", "_bz2.lib", "ffi.lib", "_ctypes.lib", "_decimal.lib", "_elementtree.lib", "_hashlib.lib", "liblzma.lib", "_lzma.lib", "cabinet.lib", "msi.lib", "_msi.lib", "_multiprocessing.lib", "_overlapped.lib", "_queue.lib", "_socket.lib", "sqlite3.lib", "_sqlite3.lib", "_ssl.lib", "tcl86t.lib", "tk86t.lib", "_tkinter.lib", "_uuid.lib", "_zoneinfo.lib", "pyexpat.lib", "select.lib", "unicodedata.lib", "winsound.lib", "Winmm.lib"])
```

Comment:
Would there be a way to tell that it's Nuitka-Python? These won't exist otherwise.

Reply:
Well, in theory they are already in a check for if it is windows or not and Nuitka - Python is the only static windows version I believe. But nonetheless we can add a check based on platform.python_implementation() here.